doc/source/getting_started/docker/linux_container.rst

```diff
@@ -0,0 +1,178 @@
+.. _ref_linux_docker:
+
+Geometry Service as a Linux Docker container
+==============================================
+
+.. contents::
+
+.. _ref_running_linux_containers:
+
+Running Linux Docker containers
+---------------------------------
+
+To run the Linux Docker container for the Geometry service, ensure that you follow
+these steps when installing Docker:
+
+.. tab-set::
+
+    .. tab-item:: Linux machines
+
+       If you are on a Linux machine, install `Docker for your distribution <https://docs.docker.com/engine/install/#server>`_
+
+    .. tab-item:: Windows/MacOS machines
+
+       #. Install `Docker Desktop for Windows <https://docs.docker.com/desktop/install/windows-install/>`_ or
+          `Docker Desktop for MacOS <https://docs.docker.com/desktop/install/mac-install/>`_.
+
+       #. **(On Windows)** When prompted for **Use WSL2 instead of Hyper-V (recommended)**, **UNTICK** this checkbox.
+          Hyper-V must be enabled to run Windows Docker containers - which you might be interested in doing in the future.
+
+       #. Once the installation finishes, restart your machine, and start Docker Desktop.
+
+Now that your Docker engine supports running Linux Docker containers, you can build or install
+the PyAnsys Geometry image.
+
+Build or install the Geometry Service image
+-------------------------------------------
+
+There are two options for users to install the PyAnsys Geometry image:
+
+* Downloading it from the :ref:`GitHub Container Registry <ref_linux_docker_ghcr>`.
+* :ref:`Building the Geometry Service Linux container <ref_linux_docker_fromscratch>`.
```

Comment:
It would be nice to not using the -ing (gerun) form of the verb put the simple verb (Download... and Build...). This is the recommendation from the Google developer documentation style guide. It makes links cleaner/easier to use.